packages/react/src/tags/Tags.tsx

```diff
@@ -1,23 +1,39 @@
-import { ReactNode } from 'react';
+import { forwardRef, ReactNode, useRef } from 'react';
 import { Box } from '../box';
-import { TagsContainer } from './TagsContainer';
+import { mergeRefs } from '../core';
 import { Tag, TagProps } from './Tag';
+import { TagsContainer } from './TagsContainer';
 import { TagsList } from './TagsList';
 
 export type TagsProps = {
 	heading?: ReactNode;
 	items: Omit<TagProps, 'children'> & { label: string }[];
 };
 
-export const Tags = ({ items, heading }: TagsProps) => (
-	<TagsContainer>
-		{heading}
-		<TagsList>
-			{items.map(({ label, ...props }, index) => (
-				<Box as="li" key={index}>
-					<Tag {...props}>{label}</Tag>
-				</Box>
-			))}
-		</TagsList>
-	</TagsContainer>
-);
+export const Tags = forwardRef<HTMLDivElement, TagsProps>(function Tags(
+	{ heading, items },
+	forwardedRef
+) {
+	const ref = useRef<HTMLDivElement>(null);
+	const listRef = mergeRefs([ref, forwardedRef]);
+	const focusOnRemove = (index: number) => {
+		const nextButtonToFocus =
+			listRef?.current &&
```

Comment:
@ChrisLaneAU What am I doing wrong here? What do I need to do to appease the TypeScript gods?

```
Property 'current' does not exist on type 'RefCallback<HTMLDivElement>'.ts(2339)
```

Reply:
The reason it's complaining is that the return value of `mergeRefs` is a function, not an object with a property of `current`. `mergeRefs` doesn't merge all refs into one, it synchronises the value across all of the refs that you pass into the array. Perhaps it should be called `syncRefs`? 

Anyway, because both the `forwardedRef` and the `ref` (from line 17) still exist, and point to the same DOM node, you can just swap `listRef` for `ref` on lines 21 and 22. It's probably best to also remove the `listRef` const and pass the function straight into the ref prop on line 26, i.e. `ref={mergeRefs([ref, forwardedRef])}`.